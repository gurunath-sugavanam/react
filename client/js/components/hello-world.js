import React from 'react';
import PropTypes from 'prop-types';

export class HelloWorld extends React.Component {

  static propTypes = {
    message: PropTypes.string,
  };

  render() {
    return <h1>Hello World! {this.props.message}</h1>;
  }

}