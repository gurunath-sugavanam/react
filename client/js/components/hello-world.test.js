import React from 'react';
import renderer from 'react-test-renderer';

import { HelloWorld } from './hello-world';

describe('hello world component tests', () => {

  test('render test', () => {

    const component = renderer.create(<HelloWorld />);
    expect(component).toMatchSnapshot();

  });

});