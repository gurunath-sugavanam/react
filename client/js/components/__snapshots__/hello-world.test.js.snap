// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello world component tests render test 1`] = `
<h1>
  Hello World! 
</h1>
`;
